src/app.js

```diff
@@ -0,0 +1,46 @@
+import express from "express";
+import homeRouter from "./routes/home.js";
+import mysql from "mysql2/promise";
+import {createProductTable, createRecipeTable} from "./database/tables.js";
+import recipeRouter from "./routes/recipeController.js";
+import productRouter from "./routes/productController.js";
+
+
+export let db;
+
+try {
+    db = await mysql.createConnection({
+        host: "localhost",
+        user: "root",
+        port: 3310,
+        password: "pass123",
+    })
+
+    await db.query(`CREATE DATABASE IF NOT EXISTS cook_assistant`);
+    await db.query(`USE cook_assistant`);
+
+    createRecipeTable(db);
```

Comment:
Application code should not create database.

Reply:
I use it to make easier, test it on your PC, but if u didn't test it, i can remove it before next PR.

Reply:
Export to  .sql file with next task

Reply:
Sql file will be executte one manually it's not needed to check it every time app starts (time wasting during boot). I'm not surethis or previous task mentioned create sql file